.github/workflows/test.yaml

```diff
@@ -0,0 +1,63 @@
+name: Test
+
+on: push
```

Comment:
Anything we need to do to lock this down to not run from forks? I could see this potentially getting abused...

I _think_ the default is to now allow access to secrets from forks, but we may just wanna verify that if we haven't already.

Reply:
CI processes will not run from forks. To trigger tests from a fork, somebody with write access to the repo has to push a commit.

Reply:
Ref: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

Reply:
Actually it looks like it's now possible to trigger it without pushing a commit. This must be a relatively new feature as it didn't used to work this way.

> If you are comfortable with running workflows on the pull request branch, return to the  Conversation tab, and under "Workflow(s) awaiting approval", click Approve and run.

Good call out.